controls/pcidss_4.yml

```diff
@@ -2045,1274 +2629,1420 @@ controls:
         - service_auditd_enabled
       controls:
       - id: 10.2.1.1
-        title: 'Audit logs capture all individual user access to cardholder
-          data.Records of all individual user access to cardholder data are captured.'
+        title: Audit logs capture all individual user access to cardholder data.
+        description: |-
+          Records of all individual user access to cardholder data are captured.
         levels:
           - base
-        status: automated
-        rules:
-          - audit_rules_unsuccessful_file_modification_creat
-          - audit_rules_unsuccessful_file_modification_rename
-          - audit_rules_unsuccessful_file_modification_open
-          - audit_rules_unsuccessful_file_modification
-          - audit_rules_unsuccessful_file_modification_ftruncate
-          - audit_rules_unsuccessful_file_modification_renameat
-          - audit_rules_unsuccessful_file_modification_truncate
-          - audit_rules_unsuccessful_file_modification_unlink
-          - audit_rules_unsuccessful_file_modification_openat
-          - audit_rules_unsuccessful_file_modification_unlinkat
-          - audit_rules_unsuccessful_file_modification_open_by_handle_at
+        status: planned
+        notes: |-
+          Differently than 10.2.1.4, this requirement is about logginh successful access to
+          cardholder data. This kind of events can easily hit performance issues and are usually
+          not necessary if a good access policy is in place. More clarification is needed about
+          this requirement.
+        related_rules:
+          # There are already many rules related to successful access, if we need to include them.
+          # e.g.:
+          - audit_rules_successful_file_modification_open
 
       - id: 10.2.1.2
-        title: 'Audit logs capture all actions taken by any individual with
-          administrative access, including any interactive use of application or system
-          accounts.Records of all actions performed by individuals with elevated
-          privileges are captured.'
+        title: Audit logs capture all actions taken by any individual with administrative access,
+          including any interactive use of application or system accounts.
+        description: |-
+          Records of all actions performed by individuals with elevated privileges are captured.
         levels:
           - base
-        status: automated
+        status: partial
+        notes: |-
+          Not all privileged commands have suid or sgid enabled. We probably want to include more
+          rules for this requirement.
         rules:
           - audit_rules_privileged_commands
```

Comment:
I agree on replacing `audit_rules_privileged_commands` by `audit_rules_suid_privilege_function`. Since the policy is not specific about commands, this rule you suggested makes more sense. Thanks.

Included in 40385e58554e370d37033b6cca9210b7832e694b

`sudo_custom_logfile` is already in `2.2.6`.